maro/simulator/scenarios/data_center/physical_machine.py

```diff
@@ -0,0 +1,65 @@
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT license.
+
+from typing import List, Set
+
+
+class PhysicalMachine:
+    """Physical machine object.
+
+    Args:
+        id (int): PM id, from 0 to N. N means the amount of PM, which can be set in config.
+        cpu_cores_capacity (int): The capacity of cores of the PM, which can be set in config.
+        memory_capacity (int): The capacity of memory of the PM, which can be set in config.
+    """
+    def __init__(self, id: int, cpu_cores_capacity: int, memory_capacity: int):
+        # Required parameters.
+        self.id: int = id
+        self.cpu_cores_capacity: int = cpu_cores_capacity
+        self.memory_capacity: int = memory_capacity
+        # PM resource.
+        self._live_vms: Set(int) = set()
+        self.cpu_allocation: int = 0
+        self.memory_allocation: int = 0
+        self._cpu_utilization: float = 0.0
+        self._cpu_utilization_series: List[float] = []
```

Comment:
no need for `` self._cpu_utilization_series`` and `` self._energy_consumption`` if we put these 2 info in snapshot_list?